sdk/appconfiguration/azure-data-appconfiguration/vcpkg/portfile.cmake

```diff
@@ -12,6 +12,9 @@ vcpkg_from_github(
     HEAD_REF main
 )
 
+file(GLOB_RECURSE unused "${SOURCE_PATH}/cgmanifest.json")
```

Comment:
Is {SOURCE_PATH} limited to folders belonging to the SDK? I wouldn't want to remove other files from the machine.